aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -78,17 +83,23 @@ public static void beforeClass() {
     clientFactory = AwsClientFactories.defaultFactory();
     s3 = clientFactory.s3();
     kms = clientFactory.kms();
+    s3Control = clientFactory.s3Control();
     bucketName = AwsIntegTestUtil.testBucketName();
+    accountId = AwsIntegTestUtil.testAccountId();
+    accessPointName = AwsIntegTestUtil.testAccessPointName();
     prefix = UUID.randomUUID().toString();
     contentBytes = new byte[1024 * 1024 * 10];
     deletionBatchSize = 3;
     content = new String(contentBytes, StandardCharsets.UTF_8);
     kmsKeyArn = kms.createKey().keyMetadata().arn();
+
+    AwsIntegTestUtil.createAccessPoint(s3Control, accessPointName, bucketName, accountId);
```

Comment:
if we are creating the access points on the fly, why do we still need those environment variables? Can't we just define the `accessPointName` in tests?

Reply:
yes , we can and these arguments are indeed not required in the present set-up can be directly accessed in the Util class itself.